### PR TITLE
Add new init function to allow disabled options without breaking backwards

### PR DESCRIPTION
**Pull request by hannabergljot**
Dropdown component can be initiated with only value and text using the init function as before. For Dropdown components with disabled options, initWithOptionsProperties can be used:)

### CODE REVIEW

File: src/Nordea/Components/Dropdown.elm [hunk 1]
```diff
@@ -98,8 +98,25 @@ simple options optionToString onInput =
...
+    in
+    Dropdown
+        { placeholder = Nothing
+        , onInput = onInput
+        , options = optionsWithDisabled
+        , optionToString = optionToString
+        , selectedValue = Nothing
+        , hasError = False
+        , variant = Standard
+        }
```
Comment (fredrikloberg) [inline src/Nordea/Components/Dropdown.elm:115]:
```suggestion
    initWithOptionProperties (List.map optionInit options) optionToString onInput
```

**Review by fredrikloberg — APPROVED** [review]
Nice solution 🤩

File: src/Nordea/Components/Dropdown.elm (hunk 1 above)
Comment (hannabergljot) [reply src/Nordea/Components/Dropdown.elm:115]: Clever😎

**Review by mathilwa — APPROVED** [review]
Nice!

**Pull request merged**